src/mockups/VaultV2.sol

```diff
@@ -254,6 +254,12 @@ contract VaultV2 {
             _setBaseCurrency(baseCurrency);
         }
         success = getFreeMargin() >= amount;
+        // Update the vault values
+        (address[] memory AllAssetAddresses, uint256[] memory AllAssetIds, uint256[] memory AllAssetAmounts) =
```

Comment:
Can we rename this to ` (address[] memory assetAddresses, uint256[] memory assetIds, uint256[] memory assetAmounts)` to be consistent with other generateAssetData's?